bin/ch/ch.cpp

```diff
@@ -671,6 +671,10 @@ HRESULT ExecuteTest(const char* fileName)
         }
     }
 Error:
+    if( fileContents != nullptr)
+    {
+        free((char*)fileContents);
```

Comment:
I added a commit to the changeset that attempts to address this, and an ASAN run after adding it came back clean. There's likely some work that could be done on the JSRT api related to this though; there doesn't seem to be a good way to, for example, handle a case where you have some data that you want to hold around needed to create the source for a serialized function with a deferred source initializer; even if we were to add the finalize callback for it, it doesn't handle the case where it's never needed, where we'd still need to free the data that would be needed in the case that the deferred initializer were to run.